submarine-server/server-core/src/test/java/org/apache/submarine/server/rest/JobManagerRestApiTest.java

```diff
@@ -137,16 +134,12 @@ public void testGetJobById() throws IOException {
         .create();
     JsonResponse<Job> jsonResponse = gson.fromJson(responseBodyAsString,
         new TypeToken<JsonResponse<Job>>(){}.getType());
-    assertEquals("Response code should be 200 ",
-        Response.Status.OK.getStatusCode(), jsonResponse.getCode());
-    assertEquals("Job id should be " + jobId, jobId,
-        jsonResponse.getResult().getJobId().toString());
+    assertEquals(Response.Status.NOT_FOUND.getStatusCode(), jsonResponse.getCode());
```

Comment:
`testGetJobById()` needs to be able to get the JOB after submitting the JOB instead of `NOT_FOUND` job, otherwise this test case is meaningless.

Reply:
This probably needs to mock the JobManager internals job list beforehand or post jobs specs to ensure the JobManager store it before the query.
@liuxunorg any better idea or help on how to mock the submarine server? The AbstractSubmarineServerTest seems is starting up a real submarine server. To me it seems not easy to mock the submarine server's internals easily.

Reply:
Hi @tangzhankun , I mocked the Submitter to resolve the Job REST test.